SwiftCharts/ChartCoordsSpace.swift

```diff
@@ -118,7 +174,18 @@ public class ChartCoordsSpace {
                 layer.rect.width + chartSettings.spacingBetweenAxesY
             }, generator: generator)
     }
-    
+
+    /**
+     Uses a generator to make axis layers from axis models. This method is used for all axes.
+
+     - parameter axisModels:    The models used to generate the axis layers
+     - parameter offset:        The offset in points for the generated layers
+     - parameter pointsCreator: TODO: not sure exactly what this effectively does
```

Comment:
Ah, okay I think I understand how this and dimIncr are intended to be used and what they do. Just to make sure: When creating axis layers for a particular dimension in a multi-axis chart, pointsCreator is a closure that takes a given screen location offset, along the variable dimension, as input and calculates the screen locations of the smallest and largest axis values. dimIncr calculates the length (width or height, depending on dimension) of the axis so that the next axis is offset by the correct amount. Is this correct?